redbot/cogs/audio/core/commands/audioset.py

```diff
@@ -1117,21 +1117,29 @@ async def command_audioset_settings(self, ctx: commands.Context):
             if global_data["use_external_lavalink"]
             else _("Disabled"),
         )
-        if is_owner and not global_data["use_external_lavalink"] and self.player_manager.ll_build:
+        if (
+            is_owner
+            and not global_data["use_external_lavalink"]
+            and self.managed_node_controller.ll_build
+        ):
             msg += _(
                 "Lavalink build:         [{llbuild}]\n"
                 "Lavalink branch:        [{llbranch}]\n"
                 "Release date:           [{build_time}]\n"
                 "Lavaplayer version:     [{lavaplayer}]\n"
                 "Java version:           [{jvm}]\n"
                 "Java Executable:        [{jv_exec}]\n"
+                "Xms:                    [{xms}]\n"
+                "Xmx:                    [{xmx}]\n"
```

Comment:
Perhaps it would make more sense to refer to these the same as in the llset commands - initial and max heapsize?